### PR TITLE
Getting profile image url s3

### PR DESCRIPTION
**Pull request by punchaal**
Added logic for getting profile image url after upload to s3.

### CODE REVIEW

**Comment by skassam21** [comment]
You should not be committing node_modules - it should be gitignored - it makes the PR look way bigger and impossible to get through

File: tuneful-client/src/CustomHooks/useForm.js [hunk 1]
```diff
@@ -37,6 +36,9 @@ const useForm = (callback) => {
     event.preventDefault()
     const { email, password } = event.target
  
+    console.log('login form submitted')
+    console.log({ email, password })
```
Comment (skassam21) [inline tuneful-client/src/CustomHooks/useForm.js:40]: remove console logs, especially password

File: tuneful-client/src/Services/token-service.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ const TokenService = {
   hasAuthToken() {
     return !!TokenService.getAuthToken()
   },
-  makeBasicAuthToken(userName, password) {
-    return window.btoa(`${userName}:${password}`)
+  makeBasicAuthToken(email, password) {
+    return window.btoa(`${email}:${password}`)
```
Comment (skassam21) [inline tuneful-client/src/Services/token-service.js:17]: are you using this or token auth at the moment?

Comment (punchaal) [reply tuneful-client/src/Services/token-service.js:17]: Just reviewed this. It's just a services file. The method exists from before when I was using basicauth. It is not used in the code anymore. Not sure why it was updated in this PR.

**Pull request merged**